spec/views/shared/_alert.html.erb_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+describe 'shared/_alert.html.erb' do
+  it 'renders message from param' do
+    render 'shared/alert', { message: 'FYI' }
+
+    expect(rendered).to have_content('FYI')
+  end
+
+  it 'renders message from block' do
+    render 'shared/alert' do 'FYI' end
```

Comment:
```suggestion
    render 'shared/alert' { 'FYI' }
```

Reply:
Hm, I'd actually used this syntax in earlier iterations, but encountered syntax errors:

```
An error occurred while loading ./spec/views/shared/_alert.html.erb_spec.rb.
Failure/Error: __send__(method, file)

SyntaxError:
  /Users/andrewmduthie/Documents/Code/identity-idp/spec/views/shared/_alert.html.erb_spec.rb:11: syntax error, unexpected '{', expecting end
      render 'shared/alert' { 'FYI' }
                            ^
  /Users/andrewmduthie/Documents/Code/identity-idp/spec/views/shared/_alert.html.erb_spec.rb:11: syntax error, unexpected '}', expecting end
  ...render 'shared/alert' { 'FYI' }
  ...                              ^
  /Users/andrewmduthie/Documents/Code/identity-idp/spec/views/shared/_alert.html.erb_spec.rb:14: syntax error, unexpected end, expecting end-of-input
    end
    ^~~
```

Admittedly I'm not entirely clear on what the specific issue is here. I'd assumed it might be trying to treat `{ 'FYI' }` as a hash? And/or parameter to `render` or the result of `render('shared/alert')`?

I'll follow-up once I've learned more 😄

Reply:
>I'll follow-up once I've learned more 😄

I think this is it: https://stackoverflow.com/a/6854346

i.e. It's interpreting as trying to provide the block as an argument to the string.

Revised solution:

```suggestion
    render('shared/alert') { 'FYI' }
```

Reply:
> Revised solution:

Updated in 564c69237.